data_handler/handlers/loan_states/zklend/settings.py

```diff
@@ -77,13 +78,13 @@ async def fetch_zklend_specific_token_settings():
     Fetch ZkLend specific token settings.
     """
     # New dict to store ZkLendSpecificTokenSettings
-    NEW_ZKLEND_SPECIFIC_TOKEN_SETTINGS: dict[str, ZkLendSpecificTokenSettings] = {}
+    new_zklend_specific_token_settings: dict[str, ZkLendSpecificTokenSettings] = {}
```

Comment:
```suggestion
    zklend_specific_token_settings: dict[str, ZkLendSpecificTokenSettings] = {}
```

Reply:
fix it 73f712a53242c2402dbad84e335c8b84ae1e3ebf